.devcontainer/devcontainer.json

```diff
@@ -10,7 +10,7 @@
       "ghcr.io/devcontainers/features/github-cli:1": {}
     },
     "forwardPorts": [9080],
-    "postCreateCommand": "yarn global add @adobe/aio-cli@latest && yarn install",
+    "postCreateCommand": "yarn global add @adobe/aio-cli@latest && yarn install && yarn build",
```

Comment:
Added to run build command for `packages/commerce-sdk-auth` after container creation